lib/broccoli/fastboot-config.js

```diff
@@ -104,7 +104,9 @@ FastBootConfig.prototype.readAssetManifest = function() {
     var assetMap = JSON.parse(fs.readFileSync(assetMapPath));
     return assetMap;
   } catch (e) {
-    // No asset map was found, proceed as usual
+    if (assetMapPath) {
```

Comment:
@tomdale changed made in https://github.com/ember-fastboot/ember-cli-fastboot/pull/262/commits/edcd3af9b284e67cf6c1ae9e46a694a3bf98b5d6